manifests/q/Quadren/Arc/Prerelease/PBv1.1.3/Quadren.Arc.Prerelease.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Quadren.Arc.Prerelease
+PackageVersion: PBv1.1.3
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/quadren/arc/releases/download/PBv1.1.3/Arc_Setup.exe
+  InstallerSha256: BAFFAB1BD31FD6303B6B2BDCB250F40935224D6866FF6DFAED7848087348339D

```

```suggestion
  InstallerSha256: BAFFAB1BD31FD6303B6B2BDCB250F40935224D6866FF6DFAED7848087348339D
  ProductCode: '{E07B6521-54F8-498E-A095-9BAEBBA98C44}_is1'
```
Although not mandatory for `.exe` installers, but helps WinGet to better match upgrades.